### PR TITLE
doc: add install instructions

### PR DESCRIPTION
**Pull request by jiaxunsongucb**
**What type of PR is this?**
/kind documentation

> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Add instructions to install the driver.
Depend on this PR: https://github.com/kubernetes-csi/csi-driver-nfs/pull/80

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -44,6 +44,10 @@ all: nfs
 sanity-test: nfs
 	./test/sanity/run-test.sh
 
+.PHONY: integration-test
```
Comment (andyzhangx) [inline Makefile:47]: does `integration-test` work now?

Comment (andyzhangx) [reply Makefile:47]: if it works, need to add to CI test.

**Review by andyzhangx — COMMENTED** [review]
/ok-to-test

File: Makefile (hunk 1 above)
Comment (jiaxunsongucb) [reply Makefile:47]: OGM, I pushed to the wrong branch. Sorry, I am still testing it.

**Pull request closed without merging**